### DISCUSSION TITLE
Livewire suddenly stopped working with Console Error : Invalid or unexpected token on port 8000

### DISCUSSION THREAD

**Discussion by onensensy** [livewire/livewire#6968] [7. Bugs (archived)]
Livewire suddenly stopped working in my project and in the console the messaage is as bellow:

``
livewire.js:7802 Uncaught SyntaxError: Invalid or unexpected token
``

I have tried to troubleshoot and find a solution by changing layout, removing and even totally. I tried to implement the Counter component and still the same thing.

I came across an issue(https://github.com/livewire/livewire/discussions/6879) with my exact same problem, but it was closed without an answer. He "solved" it by moving to a fresh laravel project which I cannot afford to do.

Assistance on this would be much appreciated.

**Comment by onensensy** [comment]
I have done some more debugging and i have found that when i host my system on port 8000, there is an issue, but when i host it on port 8001, there seems to be no issue

Heres on port 8001
![image](https://github.com/livewire/livewire/assets/83227737/06f79700-f6f4-4205-b32b-39ec2d62bb5c)

Here is  on port 8000
![image](https://github.com/livewire/livewire/assets/83227737/9567bc0d-3e91-45fd-b019-f88f39c251c2)

Console:
![image](https://github.com/livewire/livewire/assets/83227737/54fb91d8-55a9-4f66-bc88-4bfe93c19e06)

**Comment by jonerickson** [comment]
Having this error as well. Running on port 80. Anyone have a solutions? Multiple posts reporting this issue with no answer.

**Comment by jonerickson** [comment]
Finally found the issue, simply just an extra space at the top of one of my PHP files. The opening PHP tag started on line 2 instead of 1.
